src/preamble.js

```diff
@@ -1918,7 +1918,7 @@ addOnPreRun(function() {
       Runtime.loadDynamicLibrary(lib);
     });
   }
-  asm['runPostSets']();
+  asm['runPostSets'] && asm['runPostSets']();
```

Comment:
please write it as `if(x) x();`

(`x && x();` is compact, but looks like something closure compiler would emit ;)